Makefile

```diff
@@ -0,0 +1,12 @@
+PORTAINER_TEMPLATES_YAML = $(shell find -name 'portainer-template.yaml' | sort)
+
+portainer/templates.json: $(PORTAINER_TEMPLATES_YAML)
+# Will wait forever miserably if the requisites are empty
```

Comment:
which requirements? :P

Reply:
just fyi, here, in the same fashion, we'll add later the step to fetch recipes stored externally. In my branch is a simple  `build.sh`

Reply:
Hi @taliaga 
if there are no requisites the grep in next lines will wait forever because no arguments, really and edge case...